### PR TITLE
Include card component

### LINKED ISSUE #62: Create a card component
Create a card component matching the structure of the screenshot below. The component should have two rows, both broken into two columns: 30% and 70%. Alignment of the elements should match the screenshot.

Please follow the instructions in [OUTREACHY.md](https://github.com/Pocket/curation-tools-frontend/blob/main/OUTREACHY.md) when developing the component.

All tasks should be completed using Typescript.

card

### PR DESCRIPTION
**Pull request by Joaquin-M2**
## Goal

Creating a card component as requested on issue #62 .

Closes #62 .

## Todos:
- [x] Create card component.
- [x] The card component should be made of two columns:
       The first one should occupy the 30% of the width of the component;
       The second one should occupy the 70% of the width of the component.
- [x] The card component should admit a prop for the path of the image (upper left cell).

## Implementation Decisions

In order to create the card component, it was used the Material UI library.

In order to adapt the components provided by Material UI to needs of this (Pocket) card component, some of the styles used by default by Material UI have been modified. For that purpose, the "makeStyles" and "createStyles" functions were used.

For a better type-safe code, TypeScript was used whenever possible and necessary (type inference was considered). E.g. creation of an interface for the props of the component, as well as defining the type of the functional component itself.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 function App() {
-  return (
-    <div className="App">
-      <!-- your component goes here to test the rendering! -->
-    </div>
-  );
+    return (
+        <div className="App">
+            {/* <!-- your component goes here to test the rendering! --> */}
+            <PocketCard />
```
Comment (kkyeboah) [inline src/App.tsx]: Could we update this component to accept an image path as a `prop`?

File: src/components/card/card.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```
Comment (kkyeboah) [inline src/components/card/card.tsx]: could we uppercase the filename?

File: src/components/card/card.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+const useStyles = makeStyles(() =>
+    createStyles({
+        root: {
+            margin: 'auto',
+            maxWidth: 500,
+            padding: 10,
+        },
+        leftColumn: {
+            width: '30%',
```
Comment (kkyeboah) [inline src/components/card/card.tsx]: nice!

File: src/components/card/card.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        },
+        rightColumn: {
+            width: '70%',
```
Comment (kkyeboah) [inline src/components/card/card.tsx]: nice!

**Review by kkyeboah — APPROVED** [review]
This is a really great start! We just have a couple of small comments

**Review by kkyeboah — CHANGES_REQUESTED** [review]
approved by accident

File: src/components/card/card.tsx (hunk 2 above)
Comment (Joaquin-M2) [reply src/components/card/card.tsx]: You mean uppercasing the 'React', right? --> import React from 'React'

If that's the case, I will need to tweak some stuff within the node_modules folder. Do you want me to proceed?

Comment (kkyeboah) [reply src/components/card/card.tsx]: No, just the filename i.e. `card.tsx` to `Card.tsx`

**Comment by Joaquin-M2** [comment]
Alright, I would say the component is ready. I also included an interface type structure for the props of the Card component.

**Review by kkyeboah — DISMISSED** [review]
This is really great work 👍. Thanks for your contribution.

Please revert the changes to App.tsx and we will merge this!

**Comment by Joaquin-M2** [comment]
> This is really great work . Thanks for your contribution.
> 
> Please revert the changes to App.tsx and we will merge this!

Ready to merge! :-)

**Review by kkyeboah — COMMENTED** [review]
Hi @Joaquin-M2, please make sure there are no differences, in the `App.tsx` file in your PR and the `main` branch of this repository.

**Comment by Joaquin-M2** [comment]
> Hi @Joaquin-M2, please make sure there are no differences, in the `App.tsx` file in your PR and the `main` branch of this repository.

[In my forked repo](https://github.com/Joaquin-M2/curation-tools-frontend/blob/main/src/App.tsx) the `App.tsx` file is not missing anything, but on the PR it is missing the importing logo line. Do you know if this is a GitHub malfunctioning, or I messed something up?

EDIT: Nevermind, I just noticed I missed something.

EDIT 2: Now it is ready to merge, I hope!

**Review by kkyeboah — APPROVED** [review]

**Pull request merged**